src/Core/src/Handlers/Border/BorderHandler.Android.cs

```diff
@@ -34,21 +34,21 @@ public override void SetVirtualView(IView view)
 			NativeView.CrossPlatformArrange = VirtualView.CrossPlatformArrange;
 		}
 
-		void UpdateContent()
+		static void UpdateContent(IBorderHandler handler)
 		{
-			_ = NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
-			_ = MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} should have been set by base class.");
-			_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
+			_ = handler.NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
+			_ = handler.MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} should have been set by base class.");
+			_ = handler.VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
```

Comment:
This validation seems to be repeated in many of the handlers. Would it make sense to dedupe it by moving it to a utility method?

Reply:
sounds good 👍

Though I think some of those aren't needed anymore and are just left over from older code. 

`VirtualView` and `NativeView` will throw exceptions if they are null when you call into them

![image](https://user-images.githubusercontent.com/5375137/154153990-3ea7ffd3-20ef-4c97-8b77-ebde1de43de0.png)

Reply:
Thanks! For now it was just a question. I will address this in one of the follow-up PR-s.